src/ui/OmniboxResultList/OmniboxResultList.ts

```diff
@@ -334,6 +335,19 @@ export class OmniboxResultList extends ResultList implements IComponentBindings
     );
     window.location.href = result.clickUri;
   }
+
+  private removeDOMChildren() {
```

Comment:
Thanks for the comment it provides good context. I am wondering if rather than removing the `coveo-result-list-container` element(s) after they have been added, would it make sense to prevent them from being added in the first place?

Reply:
This is what I tried at first but after multiple failed attempts where the omniboxResultList just stopped working, I tried this solution and it worked.

I if necessary, I can continue to check but I'm pretty sure it would require changes in the resultList

Reply:
Nice thing adding comments to make stuff clearer! 
-----
dissapears -> disappear!
Watch the punctuation too